### ISSUE TITLE
Could not find Git on the PATH - not recognized on windows

### ISSUE THREAD

**Issue by fernfab** [TypeFox/find-git-exec#11]
Message: "Could not find Git on the PATH"

OS - Windows 10 Pro
---
Git binaries on path:
C:\Users\<user>\AppData\Local\Programs\Git\bin

Using git Bash:
which
/mingw64/bin/git

**Comment by kittaakos** [comment]
Thanks for logging, @fernfab! Please confirm, does the Git feature work out of the box in VS Code?

**Comment by mcgordonite** [comment]
I see a similar problem when git can only be found on the PATH, e.g. when extracting the portable distribution of git for Windows somewhere on disk and adding it to the PATH.

In that case, the path returned from find-git-exec is just the string "git", rather than an absolute path as in all other cases. It stems from this line: https://github.com/TypeFox/find-git-exec/blob/master/src/find-git-exec.ts#L111.

VS Code git integration works fine for me with this set up.

You could use `where.exe` to convert it to an absolute path in this case, but I don't know if that's available before Windows 10.

**Comment by kittaakos** [comment]
Thank you for help, @mcgordonite.

> VS Code git integration works fine for me with this set up.

I am going to fix the issue and release a new version in the next days. Please bear with me.

**Comment by kittaakos** [comment]
For the record; I am going to add a dependency to [`which`](https://www.npmjs.com/package/which) and do exactly [what VS Code does](https://github.com/microsoft/vscode/blob/557cf012ca0d1e25366af58185de5221d47b39a3/extensions/git/src/git.ts#L116-L119).

**Comment by kittaakos** [comment]
@mcgordonite, can you please help to verify my changes? I have created a PR with the proposed fix: #12 
Can you please try the following steps from a Git bash?
```
rm -rf ./find-git-exec/ \
&& git clone --single-branch -b GH-11 https://github.com/TypeFox/find-git-exec.git \
&& yarn --cwd ./find-git-exec \
&& yarn --cwd ./find-git-exec build \
&& yarn --cwd ./find-git-exec test
```
Verifying it from a `PowerShell` and/or a `CMD.EXE` would be a great help too 😊Once you have cloned and built from a Git Bash, it would be sufficient to run `yarn test` from the other shells. Thank you!
